frontend/components/Navbar.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div w-full h="10vh" bg-black>
```

Comment:
I don't like black background, because it sometimes has diffferent color than real background, maybe make it gradient with smooth transition to black?

Reply:
well ui designer it to be black background, so I did it like this
gradient from which color to black?

Reply:
> well ui designer it to be black background, so I did it like this
hihi
> gradient from which color to black?
from transparent to black. like 0% - transparent, 15% - black, 100% - black